apps/useradmin/src/useradmin/templates/change_password.mako

```diff
@@ -64,72 +67,30 @@ ${ layout.menubar(section='users') }
     </form>
   </div>
 </div>
-
-<script src="${ static('desktop/js/hue.routie.js') }" type="text/javascript" charset="utf-8"></script>
-<script>
-  routie.setPathname('/useradmin');
-</script>
-
-
 <script type="text/javascript">
-
 $(document).ready(function(){
  var currentStep = "step1";
-
- routie({
-    "step1":function () {
-      showStep("step1");
-    }
-  });
-
-  function showStep(step) {
-    currentStep = step;
-    if (step != "step1") {
-      $("#backBtn").removeClass("disabled");
-    } else {
-      $("#backBtn").addClass("disabled");
-    }
-
-    if (step != $(".stepDetails:last").attr("id")) {
-      $("#nextBtn").removeClass("disabled");
-    } else {
-      $("#nextBtn").addClass("disabled");
-    }
-
-    $("a.step").parent().removeClass("active");
-    $("a.step[href='#" + step + "']").parent().addClass("active");
-    $(".stepDetails").hide();
-    $("#" + step).show();
-  }
-
-  function validateStep(step) {
-    var proceed = true;
-    $("#" + step).find("[validate=true]").each(function () {
-      if ($(this).val().trim() == "") {
-        proceed = false;
-        routie(step);
-        $(this).parents(".control-group").addClass("error");
-        $(this).parent().find(".help-inline").remove();
-        $(this).after("<span class=\"help-inline\"><strong>${ _('This field is required.') }</strong></span>");
+  var $editUserComponents = $('#editUserComponents');
+  % if is_embeddable:
+  $editUserComponents.find('#editForm').attr('action', window.location.pathname.substr((window.HUE_BASE_URL + '/hue').length).replace(/\/$/, ''));
+  $editUserComponents.find('#editForm').ajaxForm({
+    dataType:  'json',
+    success: function(data) {
+      if (data && data.status == -1) {
+        renderUseradminErrors(data.errors);
```

Comment:
You'll need to add this functions as well.

Reply:
Hi @JohanAhlen , the function `renderUseradminErrors` are already part layout.mako and we are importing that in our change_password.mako. dont think this is required.: 
https://github.com/cloudera/hue/blob/master/apps/useradmin/src/useradmin/templates/layout.mako#L107

Reply:
Great, thanks!